R/pages.R

```diff
@@ -0,0 +1,27 @@
+# Constants
+
+#' @export
+PAGE_404_ADDRESS <- "404"
+
+#' 404 page
+#'
+#' The page which appear when address of visited page is wrong.
```

Comment:
I think we need to use common nomenclature. For now, I see: address, route, path and page all used in similar context.

Reply:
Yeah, true, thanks for spotting it!